chianft/util/mint.py

```diff
@@ -439,7 +440,7 @@ async def submit_spend_bundles(
             launcher_ids = [
                 coin.name().hex()
                 for coin in sb.removals()
-                if coin.puzzle_hash == LAUNCHER_PUZZLE_HASH
+                if coin.puzzle_hash == INTERMEDIATE_LAUNCHER_MOD_HASH
```

Comment:
I don't immediately see how this is correct.  Maybe you could explain, or someone already familiar with it should review this.